src/content/developers/tutorials/erc20-with-safety-rails/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: ERC-20 with Safety Rails
+description: How to help people avoid silly mistakes
+author: Ori Pomerantz
+lang: en
+sidebar: true
+tags: ["erc-20"]
+skill: beginner
+published: 2022-08-15
+---
+
+## Introduction {#introduction}
+
+One of the great things about Ethereum is that there is no central authority that can modify or undo your transactions. One of the great problems with Ethereum is that ther is no central authority with the power to undo user mistakes or illicit transactions. In this article you learn about some of the common mistakes that users commit with [ERC-20](/developers/docs/standards/tokens/erc-20/) tokens, as well as how to create ERC-20 contracts that help users to avoid those mistakes, or that give a central authority some power (for example to freeze accounts).
+
+Note that while we will use the [OpenZeppelin ERC-20 token contract](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/token/ERC20), this article does not explain it in great details. You can find this information [here](/developers/tutorials/erc20-annotated-code).
+
+If you want the complete source code, you can get it [here](./ERC20-safety-rails-Remix.zip) as a [Remix workspace](https://remix.ethereum.org/).
```

Comment:
Hmmm not keen about asking people to download something here. Can't we just share a direct link to a remix workspace?

Reply:
I don't know how to do that, but I replaced it with instructions to clone a github repository.